src/main/java/racingcar/domain/RacingProcess.java

```diff
@@ -0,0 +1,30 @@
+package racingcar.domain;
+
+import camp.nextstep.edu.missionutils.Randoms;
+import java.util.ArrayList;
+
+public class RacingProcess {
+    public static void GoForwardRandom(Car car){
+        int randomnumber = Randoms.pickNumberInRange(0,9);
+        if(randomnumber >= 4){
+            car.plusProgress();
```

Comment:
```suggestion
            car.move();
```
변수의 상태를 변경한다는 것을 알리지 않고, 객체에 메시지를 던지려면 더 의미있는 메서드명을 사용하는 게 좋습니다.

Reply:
메소드를 작성할 때 한번 더 생각해보게 된것같습니다.